fabfile/setup.py

```diff
@@ -123,6 +163,16 @@ def new():
     execute(load_orchestration_and_requirements)
     execute(move_vagrantfile_to_project_dir)
     execute(create_pem_file)
+    try:
+        environment = environments[raw_input("""
```

Comment:
Are we looking for the user to input `Development` here? Would be a lot more friendly for them to just have to say `1` and you do the translation I think. Fabric/python might have a helper method to do this too.